.github/ISSUE_TEMPLATE/transfer-repo-to-this-organization.yaml

```diff
@@ -0,0 +1,82 @@
+name: Transfer repository to this organization
+description: Request to transfer a repository you own to this organization
+title: "[Transfer]: Repo-name"
+labels: ["transfer"]
+assignees:
+  - ''
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this form!
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please write a short description of the feature(s) the repository is providing
+  - type: input
+    id: code-repo
+    attributes:
+      label: Code Repository
+      description: What is the URL to your code repository on GitHub?
+      placeholder: e.g. https://github.com/org-name/project-name
+    validations:
+      required: true
```

Comment:
the procedure for non-github repos is probably to migrate to a repo in their personal namespace first, and then fill out the form. Should we document that in the readme? Also, should we allow support requests for that step?

Reply:
let's address this in a separate PR